### PR TITLE
aur: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
18.5

### CODE REVIEW

**Comment by Managor** [comment]
I'll add more commands as I learn more about this tool. For now I'll settle with merging just these two.

File: pages/linux/aur.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# aur
+
+> Build packages from AUR and manage local repositories.
+> A locar repository needs to be set in /etc/pacman.conf and vifm needs to be installed for this to fully function.
```
Comment (vitorhcl) [inline pages/linux/aur.md:4]:
```suggestion
> Note: A local repository needs to be set in `/etc/pacman.conf` and `vifm` needs to be installed for this to fully function.
```
From [style guide#special-cases](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases):
> Technical wording on description lines should use the `backtick` syntax.
> Use backticks on the following:
> 
> - Paths, e.g. `package.json`, `/etc/package.json`.
> - Extensions, e.g. `.dll`.
> - Commands, e.g. `ls`.
> - Standard streams: `stdout`, `stdin`, `stderr`. **Do not** use the full names (e.g. standard output).

**Review by vitorhcl — APPROVED** [review]
Thank you for your contribution! LGTM after my suggestions

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
> Hey btw. Is it ok to leave those shortform brackets now that I changed the options to longform?

Yeah, you can leave the mnemonics as it is, to highlight the short options too.

**Pull request merged**